build.bash

```diff
@@ -272,8 +303,8 @@ if [[ $hw_platform == "pi-raspios32" ]] || [[ $hw_platform == "pi-raspios64beta"
 
   # Prerequisites
   echo_process "Checking prerequisites... "
-  REQ_COMMANDS="git curl unzip crc32 dos2unix xz"
-  REQ_PACKAGES="git curl unzip libarchive-zip-perl dos2unix xz-utils"
+  REQ_COMMANDS="git curl wget unzip crc32 dos2unix xz"
```

Comment:
one day we should agree on either curl OR wget but not today

Reply:
We did for just the build.bash script, but our other code that is used still uses both standards.